src/app/shared/pipes/parse-date.pipe.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({
+  name: 'parseDate',
+})
+export class ParseDatePipe implements PipeTransform {
+  public transform(value: any): any {
+    if (!value) {
+      return value;
+    }
+
+    const date = new Date(value);
+    if (date.toString() === 'Invalid Date') {
```

Comment:
moment has function to test if date is valid, but maybe this is also good solution

Reply:
Thanks. I will probably be better to use moment for this since it can parse more variant of ISO date format.